packages/ember-glimmer/lib/helpers/-class.js

```diff
@@ -0,0 +1,25 @@
+import { InternalHelperReference } from '../utils/references';
+import { dasherize } from 'ember-runtime/system/string';
+
+function classHelper({ positional }) {
+  let path = positional.at(0);
+  let className = positional.at(1);
+  let value = path.value();
+
+  if (value === true) {
+    return dasherize(className.value());
```

Comment:
The behavior today in Ember is if `{{foo-bar class=fooBar}}` is executed and `fooBar` evaluates to `true` we have to reflect on the binding e.g. `fooBar` and turn the binding name to `foo-bar`.